tests/teardown.sh

```diff
@@ -4,4 +4,6 @@
 . "$WORKSPACE/.tox_vars"
 cd "$CEPH_ANSIBLE_SCENARIO_PATH" || exit 1
 vagrant destroy --force
+# see https://github.com/ceph/ceph-container/issues/1048
+sudo chown -R "$(whoami)" ./staging || true
```

Comment:
why `|| true`?

Reply:
@GregMeno If you ping me on a bug I'd love having the time to wakeup and comment before you actually merge without any feedback.

The teardown is about cleaning and the makefile have some cleaning routines. So a sudo make clean should be enough.